package.json

```diff
@@ -50,7 +50,10 @@
     "datocms-listen": "^0.1.15",
     "datocms-structured-text-utils": "^2.0.4",
     "get-video-id": "^3.6.5",
+    "github-slugger": "^2.0.0",
     "globby": "^13.2.2",
+    "hast-util-from-html": "^2.0.3",
```

Comment:
Some of these new dependencies are interfering with the build. Will have to do some triage first because I don't yet know what the exact problem is.